conda_gitenv/__init__.py

```diff
@@ -1,6 +1,19 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from conda import __version__ as CONDA_VERSION
+
 from ._version import get_versions
+
+
+def _parse_conda_version_major_minor(string):
+    return string and tuple(int(x) for x in (string.split('.') + [0, 0])[:2]) or (0, 0)
```

Comment:
We should use distutils for this purpose. https://stackoverflow.com/questions/11887762/compare-version-strings-in-python